AUTHENTICATION.md

```diff
@@ -65,7 +78,7 @@ SERVICE_ACCOUNT_NAMESPACE=crossplane-system
 SERVICE_ACCOUNT_NAME=provider-aws-<YOUR-SERVICE-ACCOUNT-EXTENSION>
```

Comment:
Can we do everything AWS-related first and then Crossplane so that there wouldn't be much context switch?

Reply:
Unfortunately no adding the annotation for the ServiceAccount requires some AWS information, but then we need to create permissions based on the name of the ServiceAccount that is created.

Reply:
I ran into the same issue trying to wrap crossplane setup in a helm chart. A simple way to do it would be to hardcode the serviceAccount name somewhere, for instance through a new `serviceAccountName` parameter next to the  `useServiceAccount` boolean: https://github.com/crossplane/provider-aws/blob/67203492ddd76bcd481683b51f6c22b92e65e182/package/crds/aws.crossplane.io_providers.yaml#L68

Reply:
@rozcietrzewiacz thanks for your thoughts here :) I think it would be more likely that we solve this by setting the `ServiceAccount` in the `ControllerConfig`

Reply:
@hasheddan Ah, if that can be done - perfect!